fishtest/fishtest/views.py

```diff
@@ -288,10 +288,10 @@ def parse_spsa_params(raw, spsa):
   params = []
   for line in raw.split('\n'):
     chunks = line.strip().split(',')
-    if len(chunks) == 0:
+    if len(chunks)==1 and chunks[0]=="": # blank line
```

Comment:
@vdbergh what about to check also for a line with only spaces? 
```python
if len(chunks)==1 and (not chunks[0] or chunks[0].isspace()): # blank line or only spaces
```

Reply:
I think this is not necessary since `line` was already stripped when defining `chunks`.